doc/user/content/versions.json

```diff
@@ -1,10 +1,10 @@
 [
     {
         "name": "Cloud",
-        "base_url": ""
+        "base_url": "/docs"
```

Comment:
So .. the docs selector doesn't populate for me even with this change.
https://preview.materialize.com/materialize/31055/

If known thing to be tinkered with this week, just let me know and I can approve this if this is an intermediate step towards the fix.

Reply:
Oh ... it might be a preview thing because I see on the current materialize/docs site, that the dropdown is indeed populated (but, as this PR fixes, the path is missing the /docs part if you switch docs).  Eh.  

Will add the approval in case it is just a preview thing.